### PR TITLE
Feature/advanced selector v2

### PR DESCRIPTION
**Pull request by AlexeySafronov**
Remasterd AdvancedSelector

- Added paging loader 
- Added Aside mode
- Added new GroupSelector
- Added new PeopleSelector
- Added Tooltips for users
- Styled and refactoring

### CODE REVIEW

File: web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js [hunk 1]
```diff
@@ -357,54 +363,40 @@ class PureAdminsSettings extends Component {
...
-                />
-                <div style={{ right: 180 }} className="advanced-selector">
-                  <AdvancedSelector
-                    displayType="dropdown"
+                <div className="people-admin_container">
+                  <Button
+                    id="people-admin-selector_button"
+                    size="medium"
+                    primary={true}
+                    label="Set people admin"
```
Comment (andreysavihin) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js]: Label text from resources

File: web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js [hunk 2]
```diff
@@ -357,54 +363,40 @@ class PureAdminsSettings extends Component {
...
-                />
-                <div style={{ right: 160 }} className="advanced-selector">
-                  <AdvancedSelector
-                    displayType="dropdown"
+                <div className="full-admin_container">
+                  <Button
+                    id="full-admin-selector_button"
+                    size="medium"
+                    primary={true}
+                    label="Set portal admin"
```
Comment (andreysavihin) [inline web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js]: Label text from resources

File: config/kafka.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
 	"kafka": {
-		"BootstrapServers": ""
+		"BootstrapServers": "localhost:9092"
```
Comment (andreysavihin) [inline config/kafka.json]: Revert changes in kafka config

File: web/ASC.Web.Common/src/components/PeopleSelector/PeopleSelector.js [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+    const user = options[+index];
+
+    if (!user) return null;
+
+    // console.log("onOptionTooltipShow", index, user);
+
+    return (
+      <div
+        style={{
```
Comment (andreysavihin) [inline web/ASC.Web.Common/src/components/PeopleSelector/PeopleSelector.js]: Use styled component

File: web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js (hunk 1 above)
Comment (AlexeySafronov) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js]: Will be fixed in another branch

File: web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js (hunk 2 above)
Comment (AlexeySafronov) [reply web/ASC.Web.Client/src/components/pages/Settings/categories/security/sub-components/admins.js]: Will be fixed in another branch

**Pull request merged**